.github/workflows/conformance.yml

```diff
@@ -32,7 +32,7 @@ jobs:
           docker run --rm -p 5000:5000 -e REGISTRY_STORAGE_DELETE_ENABLED=true -idt "registry:local"
       -
         name: Run OCI Distribution Spec conformance tests
-        uses: opencontainers/distribution-spec@main
+        uses: opencontainers/distribution-spec@v1.0.1
```

Comment:
We should make that action be "go module" aware, so that it can look at the version we vendor 😅